### PR TITLE
Correct ssh key for pablogsal

### PR DESCRIPTION
**Pull request by pablogsal**
For some unknown reason the previous ssh key did not work correctly. Thanks to @zware we managed to debug this and we checked that this keys work. This PR updates the key in the json file.

Sorry everyone for the inconvenience.

### CODE REVIEW

File: data_bags/users/pablogsal.json [hunk 1]
```diff
@@ -3,6 +3,6 @@
...
+    "sh-rsa AAAAB3NzaC1yc2EAAAADAQABAAACAQC34fbzw9gfpohw0zAAyPGkWe4PbZ88p2WJJN/trAoorrHrbEEPhyDyfwEhCQezp2p3vXck/y00YayvamsYFTIppNgjlwf8coezJ8O5yYBVNGkUm7V9L270M9rLks60OvZjys2p9vCRqApagdG8Nb8H91Y48R1dnYA+ewpqhT7+5AAv9muenRPnrc12rbWQC/tiVk6Zn0NHj0UWnbQZtFWoUIZ1PMdPo7wFaxGsKqVVvN6M2Bclv9FiJzNie4x4ENejqLGOJY+S/w8aVcWgK4URaEX1j13wt+4hPDKLvyPeYrLz0CIadcE6OIxtCjNNgaCFW6f0ETQUOlraTs7ybNYKkIiSU36myiV0NzQxGPXK6VUvNxyfmb/fTzbMjZBYxkKVPOFN2YbCOMoI3WOOZqWbnqiNUVXEUHbvB4L6KP+/EP0oq+oDBqKd5lXVaNoLiJBXCa14dF5N3nIeGlaq5isf5j1xsDgEFNAwLpeEVt3yoNUMUBO05cN/veRHv/quYxwES9Lg2wiaUTvEpfDQxLi99p/9jrepEHgkh61OyiUWAJE8ZCr6/MzaMO4RYRxTxEpx7057JCOp6IdFWOjcDIZi74ZtMCpEcWT81DXN7IC72ytTcpiGfWigmWps4Wa5hr525RM/zki8BlLmuOuR94TNb7wUImq7Y+/umaliJEpgTw== cardno:000605651315"
```
Comment (zware) [inline data_bags/users/pablogsal.json]: Looks like a leading `s` missing.

Comment (pablogsal) [reply data_bags/users/pablogsal.json]: Thanks @zware I corrected this and update the file with the rest of my keys.

File: data_bags/users/pablogsal.json [hunk 2]
```diff
@@ -3,6 +3,9 @@
...
+    "ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAACAQC34fbzw9gfpohw0zAAyPGkWe4PbZ88p2WJJN/trAoorrHrbEEPhyDyfwEhCQezp2p3vXck/y00YayvamsYFTIppNgjlwf8coezJ8O5yYBVNGkUm7V9L270M9rLks60OvZjys2p9vCRqApagdG8Nb8H91Y48R1dnYA+ewpqhT7+5AAv9muenRPnrc12rbWQC/tiVk6Zn0NHj0UWnbQZtFWoUIZ1PMdPo7wFaxGsKqVVvN6M2Bclv9FiJzNie4x4ENejqLGOJY+S/w8aVcWgK4URaEX1j13wt+4hPDKLvyPeYrLz0CIadcE6OIxtCjNNgaCFW6f0ETQUOlraTs7ybNYKkIiSU36myiV0NzQxGPXK6VUvNxyfmb/fTzbMjZBYxkKVPOFN2YbCOMoI3WOOZqWbnqiNUVXEUHbvB4L6KP+/EP0oq+oDBqKd5lXVaNoLiJBXCa14dF5N3nIeGlaq5isf5j1xsDgEFNAwLpeEVt3yoNUMUBO05cN/veRHv/quYxwES9Lg2wiaUTvEpfDQxLi99p/9jrepEHgkh61OyiUWAJE8ZCr6/MzaMO4RYRxTxEpx7057JCOp6IdFWOjcDIZi74ZtMCpEcWT81DXN7IC72ytTcpiGfWigmWps4Wa5hr525RM/zki8BlLmuOuR94TNb7wUImq7Y+/umaliJEpgTw== cardno:000605651315",
```
Comment (zware) [inline data_bags/users/pablogsal.json]: Looks like this one and the one on line 8 are duplicate; it should be safe to remove either one.

**Comment by ewdurbin** [comment]
should be picked up in 30 minutes.

**Pull request merged**